src/main/java/Remy/Ui.java

```diff
@@ -0,0 +1,123 @@
+package remy;
+
+import remy.task.Task;
+import remy.task.TaskList;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    public final Scanner scanner;
+    private static String divider = "____________________________________________________________\n";
+    private static String shortDivider = "_____________\n";
+    private static String welcomeMessage =
```

Comment:
You could prefix `final` to these variables that I assume are meant to be constants? If so, do remember to change their names to `DIVIDER`, `SHORT_DIVIDER` and `WELCOME_MESSAGE` respectively to align with the coding standards.